pkg/console/subresource/consoleserver/types.go

```diff
@@ -29,9 +29,14 @@ type Config struct {
 	I18nNamespaces           []string          `yaml:"i18nNamespaces,omitempty"`
 	Proxy                    Proxy             `yaml:"proxy,omitempty"`
 	ManagedClusterConfigFile string            `yaml:"managedClusterConfigFile,omitempty"`
+	HubClusterInfo           `yaml:"hubClusterInfo,omitempty"`
 	Telemetry                map[string]string `yaml:"telemetry,omitempty"`
 }
 
+type HubClusterInfo struct {
```

Comment:
not really sure about this struct, if we wanna have a higher abstraction here, eg. `MultiClusterInfo`, which would contain all configuration that we might add down the road as well as `HubClusterInfo`.

Thoughts ?